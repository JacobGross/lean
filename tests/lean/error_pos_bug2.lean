inductive fibrant [class] (T : Type) : Type :=
mk : fibrant T

inductive Fib : Type :=
mk : ΠA : Type, fibrant A → Fib

namespace Fib

definition type [coercion] (F : Fib) : Type := Fib.rec_on F (λA f, A)

definition is_fibrant [instance] (F : Fib) : fibrant (type F) := Fib.rec_on F (λA f, f)

end Fib
-- open Fib
-- Path

inductive path {A : Fib} (a : A) : A → Type :=
idpath : path a a
