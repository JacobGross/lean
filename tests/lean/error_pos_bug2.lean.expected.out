error_pos_bug2.lean:17:37: error: type expected at
  A
